build

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -eu
+
+export GO111MODULE=on
+
+NAME=fcct
+export GOFLAGS=-mod=readonly
+
+eval $(go env)
+if [ -z ${BIN_PATH+a} ]; then
+	BIN_PATH=${PWD}/bin/${GOARCH}
+fi
+
+export CGO_ENABLED=1
```

Comment:
minor: let's group all the exports together at the top

Reply:
fixed.